src/bluesearch/embedding_models.py

```diff
@@ -59,7 +60,7 @@ def preprocess(self, raw_sentence):
         """
         return raw_sentence
 
-    def preprocess_many(self, raw_sentences):
+    def preprocess_many(self, raw_sentences: list[str]) -> list[Any]:
```

Comment:
Can't we say for sure that the returned type is going to be `list[str]`?

Reply:
corrected